hathor/transaction/storage/transaction_storage.py

```diff
@@ -541,6 +558,41 @@ def get_all_sorted_txs(self, timestamp: int, count: int, offset: int) -> Transac
         """
         raise NotImplementedError
 
+    def add_value(self, key: str, value: str) -> None:
```

Comment:
It is missing a unittest for `add_value` and `remove_value`.

Reply:
Actually there is a test (`test_full_verification_attribute` on `tests/tx/test_tx_storage.py`) that tests the methods using `add_value` and `remove_value`.

Do you think it would be better to have another one that directly calls `add_value` and `remove_value`?

Reply:
I think it is ok if you've tested trying to delete a non-existing key and also getting a non-existing key.

Reply:
I've added another test to test the add/remove/get methods. Can you see if it's good?
https://github.com/HathorNetwork/hathor-core/pull/89/commits/afafc19673aa288fd4ed24ce0c114e12efa9975a

Reply:
Great! Thanks!